core/src/test/scala/unit/kafka/admin/TopicCommandWithAdminClientTest.scala

```diff
@@ -736,7 +735,6 @@ class TopicCommandWithAdminClientTest extends KafkaServerTestHarness with Loggin
       topicService.describeTopic(new TopicCommandOptions(Array("--topic", testTopicName))))
     val rows = output.split("\n")
     assertEquals(2, rows.size)
-    rows(0).startsWith(s"Topic:$testTopicName\tPartitionCount:1")
```

Comment:
Maybe `assertTrue(rows(0).startsWith(s"Topic:$testTopicName\tPartitionCount:1"))`?

Reply:
Thanks for your commont. Due to the introduction of topicId, this will cause assert to fail. We can’t get the topicId in advance, so we can use `assertTrue(rows(0).startsWith(s"\tTopic: $testTopicName"))`